README.md

```diff
@@ -106,13 +106,8 @@ acceptable.
 ### API
 
 ```webidl
-enum EffectState {
-   "disabled",
-   "enabled"
-}
-
 dictionary MediaEffectInfo {
```

Comment:
Is the idea that blur-specific properties would be added to a blur-specific dictionary?

The reason for the dictionary could use some justification - i.e. what are blur properties, like intensity, that we would put in it in the future.